astilectron.go

```diff
@@ -44,6 +44,11 @@ const (
 	EventNameAppTooManyAccept = "app.too.many.accept"
 )
 
```

Comment:
Could you remove the `TCP` references in this file (constants, options, comments, etc.). Simply look for "TCP" and remove it when it shouldn't be here anymore since it won't be TCP every time